libs/cardano-ledger-conformance/test/Test/Cardano/Ledger/Conformance/Imp.hs

```diff
@@ -108,8 +118,30 @@ spec =
   withImpInit @(LedgerSpec Conway) $
     modifyImpInitProtVer @Conway (natVersion @10) $
       modifyImpInitExpectLedgerRuleConformance testImpConformance $ do
-        describe "Basic imp conformance" $
+        describe "Basic imp conformance" $ do
           it "Submit constitution" $ do
             _ <- submitConstitution @Conway SNothing
             passNEpochs 2
+          xit "P1 reference scripts must be witnessed" $ do
+            kh <- freshKeyHash
+            (_, addr) <- freshKeyAddr
+            let
+              timelock = fromNativeScript $ RequireSignature @Conway kh
+              txOut =
+                BabbageTxOut
+                  addr
+                  (inject $ Coin 15_000_000)
+                  NoDatum
+                  (SJust timelock)
+            tx0 <-
+              submitTx $
+                mkBasicTx mkBasicTxBody
+                  & bodyTxL . outputsTxBodyL .~ SSeq.singleton txOut
+            let
+              txIn = mkTxInPartial (txIdTx tx0) 0
+              tx1 =
+                mkBasicTx mkBasicTxBody
+                  & bodyTxL . referenceInputsTxBodyL .~ Set.singleton txIn
+            res <- trySubmitTx tx1
+            res `shouldSatisfyExpr` isLeft
```

Comment:
This expectation is incorrect, since submission of this transaction is indeed expected to succeed. I would instead use a timelock script that can't be satisfied (eg. `AnyOf []`) and check that the transaction still succeeds
```suggestion
```